roles/filetree_create/README.md

```diff
@@ -249,6 +251,44 @@ This example will export all object but some with modifications:
 ...
 ```
 
+## Usage example for the `secrets_as_variables` feature
+
+To let the credentials and the users to be exported and imported 'as is', without any modification, the sensitive data (that can't be exported through the API) can be abstracted to extra vars (or variable's file) and vaulted. Those variables can be referenced at the original objects' code, so they can be imported without any manual modification. To clarify the described scenario, the following output shows the exported object for a gateway user, using the `secrets_as_variable` feature:
+
+```console
```

Comment:
Please add this as a sample playbook (or snippet) rather than command line. We wouldn't encourage people to run it like this, plus this is really hard to read/

Reply:
Ok, I think I got your idea :-)